CHANGELOG.md

```diff
@@ -19,6 +19,14 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [3.0.1](https://github.com/Nike-Inc/Willow/releases/tag/3.0.1)
+
+#### Updated
+
+- Fix compile error for Swift 3.2/4.0.
+- Rename Network example framework to WebServices to avoid a name collision with an iOS 11 private framework with the same name.
+- Set SWIFT_VERSION to 3.2 in all targets. All targets still build with Swift 3.1/Xcode 8.3.x.
```

Comment:
These should all probably be written in past tense. I'd also probably change the `Fix compile error` to something more like: `Updated source and test code to compile against Swift 3.2 and 4.0.`.